src/main/kotlin/com/example/fhirvalidator/configuration/SecurityConfiguration.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.fhirvalidator.configuration
+
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
+
+@Configuration
+@EnableWebSecurity
+class SecurityConfiguration : WebSecurityConfigurerAdapter() {
+    override fun configure(http: HttpSecurity) {
+        http.authorizeRequests().anyRequest().permitAll().and().csrf().disable()
```

Comment:
Annoyed that we have to disable auth manually. It's enabled as soon as you bring in the oauth2 client. Wonder if there's a better way to disable the stuff we don't need